client/layout/masterbar/item.tsx

```diff
@@ -137,6 +137,7 @@ class MasterbarItem extends Component< MasterbarItemProps > {
 								{ item.label }
 							</a>
 						) }
+						{ ! item.onClick && ! item.url && item.label }
```

Comment:
It may be better to wrap them in a `<span />` for better "consistency" and potentially less layout differences with the other item types (`<button />` and `<a />`) ?